website/docs/compilebuildoutputs.md

```diff
@@ -7,8 +7,8 @@ compilebuildoutputs "value"
 ### Parameters ###
 
 `value` one of:
-* `on`  - needs documentation.
-* `off` - needs documentation.
+* `on`  - generated file should be compiled.
```

Comment:
General thought: We have a mix between "on"/"off" and "On"/"Off" throughout the docs. Should we normalize the documentation to one or the other?